care/utils/tests/test_utils.py

```diff
@@ -477,3 +480,44 @@ def get_facility_representation(self, facility):
                 },
                 **self.get_local_body_district_state_representation(facility),
             }
+
+    def create_patient_note(
+        self, patient=None, note="Patient is doing find", created_by=None, **kwargs
+    ):
+        data = {
+            "facility": patient.facility or self.facility,
+            "note": note,
+        }
+        data.update(kwargs)
+
+        patientId = patient.external_id
+
+        refresh_token = RefreshToken.for_user(created_by)
+        self.client.credentials(
+            HTTP_AUTHORIZATION=f"Bearer {refresh_token.access_token}"
+        )
+
+        self.client.post(f"/api/v1/patient/{patientId}/notes/", data=data)
+
+    @classmethod
+    def create_notification(cls, intended_for=None, **kwargs):
+        users = User.objects.exclude(username=cls.user.username)
```

Comment:
Avoid directly referring to cls.user as this can cause unexpected results instead pass the required filters as an argument